frontend/src/routes/Upload.tsx

```diff
@@ -680,6 +695,57 @@ const MetaDataEdit: React.FC<MetaDataEditProps> = ({ onSave, disabled, knownRole
     const titleFieldId = useId();
     const descriptionFieldId = useId();
     const seriesFieldId = useId();
+    const [lockedAcl, setLockedAcl] = useState<Acl | null>(null);
+    const [aclError, setAclError] = useState<ReactNode>(null);
+    const [aclLoading, setAclLoading] = useState(false);
+    const aclEditingLocked = !!lockedAcl || aclLoading || !!aclError;
+
+    const fetchSeriesAcl = async (seriesId: string): Promise<Acl | null> => {
+        const data = await fetchQuery<UploadSeriesAclQuery>(
+            environment,
+            SeriesAclQuery,
+            { seriesId }
+        ).toPromise();
+
+        if (!data?.series?.acl) {
+            return null;
+        }
+
+        return mapAcl(data.series.acl);
+    };
+
+    const onSeriesChange = async (data: { opencastId?: string }) => {
+        setAclError(null);
+
+        if (!data?.opencastId) {
+            setLockedAcl(null);
+            resetField("acl");
```

Comment:
Do we really want to reset it in this situation? I would be fine if the ACL that were previously configured would re-eappear when the series is cleared. Not sure whats more useful...

Edit: oh especially when `CONFIG.lockAclToSeries` is not set, then this means that my configured ACLs (which I was able to edit with a series selected) get discarded when I clear the series, right?